app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>All Ingredients</h1>
+
+<% @ingredients.each do |ingredient| %>
+  <div id="ingredient-<%= ingredient.id %>">
+    <h3> <%= ingredient.name %>: <%= ingredient.cost %> </h3>
+<% end %>
```

Comment:
Usually when I see a `<div>` with an `id` then I am expecting to see some `within` blocks inside of a test. I don't think that was the case here but still a great use of the `<div>` element. Just don't forget the closing tag.